includes/class-indieauth-authorization-endpoint.php

```diff
@@ -78,6 +78,8 @@ public function scopes( $scope = 'all' ) {
 			'block'    => __( 'Allows the application to block and unlock users', 'indieauth' ),
 			'channels' => __( 'Allows the application to manage channels', 'indieauth' ),
 			'save'     => __( 'Allows the application to save content for later retrieval', 'indieauth' ),
+			// Profile
+			'profile'  => __( 'Returns a complete profile to the application. Without this only a display name, avatar, and url will be returned.', 'indieauth' ),
```

Comment:
The array items descriptions above line 82 do not end in a period and line 82 string does. Maybe leave off the trailing period in line 82 description for consistency?